app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -905,6 +907,99 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_forTwoWrongAnswers_revealHint() {
```

Comment:
Facing same issues related to tests: `No tests were found`. 
Mostly some issue with my setup work.

Reply:
I run the test cases and they are failing for me on espresso and robolectric.

Reply:
@anandwana001 are these 3 tests passing for espresso?

Reply:
@anandwana001 Actually the problem is with hints&Solutions while running the tests you might have noticed that for the state with hints, the hint is revealed without any interaction. The StateFragmentTestActivity have some issue with hints&Solutions

Reply:
Resolving this as the tests were removed.